crates/core/src/protocols/sumcheck/v3/bivariate_product.rs

```diff
@@ -119,6 +118,7 @@ where
 			.iter()
 			.map(|multilin| multilin.const_slice())
 			.collect::<Vec<_>>();
+
```

Comment:
There are a lot of these random newline corrections in this diff. Please try to avoid that.